main.py

```diff
@@ -11,7 +11,7 @@
 from dotenv import load_dotenv
 from discord.ext import commands
 from discord_slash import SlashCommand
-from discord_components import DiscordComponents, Button, Select, SelectOption, message
+from discord_components import DiscordComponents#, Button, Select, SelectOption, message
```

Comment:
Don't leave comments in like this. If it can be removed, remove them.

Reply:
Will remove